CoinDock-api/app/Models/V1/Coin.php

```diff
@@ -22,6 +22,22 @@ public function wallets(){
         return $this->hasMany(Wallet::class);
     }
 
+    public function acceptedAssets(){
+        $acceptedAssets = $this::whereStatus(1)->get();
+        return $acceptedAssets;
+    }
+
+    //Conversions that we are accepting
+    public function currencyConversions(){
+        $acceptedConversions = $this::whereStatus(1)->whereIsCrypto(0)->get();
```

Comment:
If we have simple conditions, please do it in controller (or)

either it should be static function 

eg: public function currencyConversion(){
                return self::whereStatusAndIsCrypto(1,0)->get();
      }

the function call should be Coin::currencyConversions()

Reply:
done